src/libstd/sys/unix/ext/fs.rs

```diff
@@ -32,8 +32,27 @@ pub trait FileExt {
     ///
     /// The current file cursor is not affected by this function.
     ///
-    /// Note that similar to `File::read`, it is not an error to return with a
+    /// Note that similar to [`File::read`], it is not an error to return with a
     /// short read.
+    ///
+    /// [`File::read`]: ../../../../std/fs/struct.File.html#read.v
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::os::unix::prelude::FileExt;
+    /// use std::fs::File;
+    ///
+    /// # use std::io;
+    /// # fn f() -> io::Result<()> {
+    /// let buf = &mut [0u8; 8];
```

Comment:
Can you make this `let mut buf = [0u8; 8];` and pass `&mut buf` to `read_at`? That way it'll match what we have in `std::io::Read`.

Reply:
Hum sure.